src/components/utils/onfidoApi.js

```diff
@@ -3,44 +3,27 @@ import Tracker from '../../Tracker'
 import forEach from 'object-loops/for-each'
 import { humanizeField } from '../utils'
 
-const errorType = (key, val) => {
-  if (key === 'document_detection') return 'INVALID_CAPTURE'
-  // on corrupted PDF or other unsupported file types
-  if (key === 'file') return 'INVALID_TYPE'
-  // hit on PDF/invalid file type submission for face detection
-  if (key === 'attachment' || key === 'attachment_content_type') return 'UNSUPPORTED_FILE'
-  if (key === 'face_detection') {
-    return val.indexOf('Multiple faces') === -1 ? 'NO_FACE_ERROR' : 'MULTIPLE_FACES_ERROR'
-  }
-}
+const formatError = ({response, status}) => ({status, response: JSON.parse(response)})
 
-const identifyValidationError = (error) => {
-  const fields = error.fields
-  for (const key of Object.keys(fields)) {
-    const val = fields[key]
-    console.log(humanizeField(key), val)
-    error = errorType(key, val[0])
-  }
-  return error
-}
 
-const serverError = ({status, response}) => {
-  Tracker.sendError(`${status} - ${response}`)
-  return 'SERVER_ERROR'
-}
-
-const handleError = (request, callback) => {
-  const response = JSON.parse(request.response)
-  console.log(request.status)
-  const error = request.status === 422 ?
-    identifyValidationError(response.error) :
-    serverError(request)
-  callback(error)
-}
-
-export const postToOnfido = ({blob, documentType, side}, captureType, token, advancedValidation, onSuccess, onError) => {
-  if (captureType === 'face') return sendFile({blob}, 'live_photos', token, onSuccess, (request) => handleError(request, onError))
-  sendFile({blob, type: documentType, side, advanced_validation: advancedValidation}, 'documents', token, onSuccess, (request) => handleError(request, onError))
+export const postToOnfido = ({blob, documentType, side}, captureType, token, advanced_validation, onSuccess, onError) => {
+  // Advanced validation on face is disabled for now, will be handled as part of the next story
```

Comment:
error when merging, will remove. And advanced validation is enabled for face, I only missed the comment.